extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/api/ui/UiResource.java

```diff
@@ -202,4 +205,11 @@ public UiContractNegotiation initiateContractNegotiation(ContractNegotiationRequ
     public UiContractNegotiation getContractNegotiation(@PathParam("contractNegotiationId") String contractNegotiationId){
         return contractNegotiationApiService.getContractNegotiation(contractNegotiationId);
     }
+    @GET
+    @Path("pages/dashboard-page/{endpoint}")
```

Comment:
the dashboard page does not have an endpoint

it returns the "Connector Endpoint" of your connector for displaying it via the UI so users can copy it.